dotCMS/src/main/java/com/dotmarketing/db/commands/UpsertCommand.java

```diff
@@ -0,0 +1,306 @@
+package com.dotmarketing.db.commands;
+
+import com.dotcms.repackage.org.apache.commons.lang.StringUtils;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.repackage.org.apache.commons.lang.StringUtils'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)